.github/workflows/integration.yml

```
@@ -1,6 +1,8 @@
 name: Integration Tests
 on:
-  push
+  push:
+    paths:
```

Keep me honest here 9 ([GitHub docs](https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#onpull_requestpull_request_targetbranchesbranches-ignore))

```suggestion
    paths-ignore:
```

---

ahh, yeap